buildScripts/gradle/publishing.gradle.kts

```diff
@@ -9,15 +9,33 @@ _java {
     withSourcesJar()
 }
 
+val version: String by rootProject
+
 _publishing {
     publications {
-        create<MavenPublication>("ExposedJars") {
+        create<MavenPublication>("exposed") {
+            groupId = "org.jetbrains.exposed"
             artifactId = project.name
-            from(project.components["java"])
+            version = version
+            from(components["java"])
```

Comment:
Just double-checking, this line doesn't need `project.` anymore, even though `artifactId` does?

Reply:
I'm not sure actually. I followed the example [here](https://www.jetbrains.com/help/space/publish-artifacts-from-a-gradle-project.html#publish-maven-artifacts-using-the-gradle-command-line-tool:~:text=from%20components.java).